src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/integration/sessions/AvailableVisitSessionsForReviewFailuresTest.kt

```diff
@@ -0,0 +1,125 @@
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.request.review.VisitorRestrictionsForReview
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.AvailableVisitSessionDto
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.SessionTimeSlotDto
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.VisitSchedulerPrisonDto
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.SessionRestriction.OPEN
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.dto.visit.scheduler.enums.UserType.PUBLIC
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsmanageprisonvisitsorchestration.service.PrisonerProfileService
+import java.time.LocalDate
+import java.time.LocalTime
+
+// tests have been moved here as prisonerProfileService needs to be mocked as we do 2 calls to get prisoner restrictions
+// but only the 2nd call needs to fail
+@DisplayName("Get available visit sessions marked for review test when prison API call to get prisoner restrictions returns an error")
+class AvailableVisitSessionsForReviewFailuresTest : IntegrationTestBase() {
+
+  @MockitoSpyBean
+  private lateinit var visitSchedulerClientSpy: VisitSchedulerClient
+
+  @MockitoSpyBean
+  private lateinit var whereAboutsApiClientSpy: WhereAboutsApiClient
+
+  @MockitoSpyBean
+  private lateinit var alertsApiClientSpy: AlertsApiClient
+
+  @MockitoBean
+  private lateinit var prisonerProfileService: PrisonerProfileService
+
+  private val prisonCode = "MDI"
+  private val prisonerId = "AA123456B"
+  private val visitSession1 = AvailableVisitSessionDto(LocalDate.now().plusDays(3), "session1", SessionTimeSlotDto(LocalTime.of(9, 0), LocalTime.of(10, 0)), OPEN)
+  private val visitSession2 = AvailableVisitSessionDto(LocalDate.now().plusDays(10), "session2", SessionTimeSlotDto(LocalTime.of(9, 0), LocalTime.of(10, 0)), OPEN)
+  private val visitSession3 = AvailableVisitSessionDto(LocalDate.now().plusDays(17), "session3", SessionTimeSlotDto(LocalTime.of(9, 0), LocalTime.of(10, 0)), OPEN)
+
+  private val visitSchedulerPrisonDto = VisitSchedulerPrisonDto(prisonCode, true, 2, 28, 6, 3, 3, 18)
+  private val visitorIds = listOf(1L, 2L, 3L)
+
+  val visitorRestrictionsForReview = VisitorRestrictionsForReview.entries.map { it.name }
+
+  @BeforeEach
+  fun setupMocks() {
+    visitSchedulerMockServer.stubGetPrison(prisonCode, visitSchedulerPrisonDto)
+    prisonerContactRegistryMockServer.stubDoVisitorsHaveClosedRestrictions(prisonerId, visitorIds = visitorIds, result = false)
+  }
+
+  @Test
+  fun `when call to get prisoner restrictions throws NOT_FOUND then all sessions are sent back with sessionForReview flag set to false`() {
```

Comment:
Is the description wrong?
```suggestion
  fun `when call to get prisoner restrictions throws NOT_FOUND then all sessions are sent back with sessionForReview flag set to true`() {
```